stellar-dotnet-sdk-test/RevokeSponsorshipTest.cs

```diff
@@ -0,0 +1,441 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework;
+using stellar_dotnet_sdk;
+using stellar_dotnet_sdk.responses;
+using stellar_dotnet_sdk.responses.results;
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
+using Claimant = stellar_dotnet_sdk.Claimant;
+
+namespace stellar_dotnet_sdk_test;
+
+[TestClass]
+public class RevokeSponsorshipTest
+{
+    private readonly Server _server = new("https://horizon-testnet.stellar.org");
+    private string _sponsoredId = "GC3TDMFTMYZY2G4C77AKAVC3BR4KL6WMQ6K2MHISKDH2OHRFS7CVVEAF";
+    private string _sponsoringId = "GARRDNS77ZSI6PPXRBWTHIVX4RS2ULVBKNJXFRV77AZUNLDUNV2NAHJA";
+
+    private Asset _assetA =
+        new AssetTypeCreditAlphaNum4("AAA", "GARRDNS77ZSI6PPXRBWTHIVX4RS2ULVBKNJXFRV77AZUNLDUNV2NAHJA");
+
+    private KeyPair _sponsoringAccount =
```

Comment:
Should we also not use the same accounts as the sponsor test? And make that test execute first? So we can just check here if the accounts exists if not we throw an exception. Is that a good idea?